RGB.NET.Devices.Wooting/Keyboard/WootingKeyboardLedMappings.cs

```diff
@@ -230,13 +230,89 @@ internal static class WootingKeyboardLedMappings
     };
 
+    private static readonly Dictionary<LedId, (int row, int column)> SixtyPercent = new()
+    {
+        { LedId.Keyboard_Escape, (1, 0) },
+        { LedId.Keyboard_1, (1, 1) },
+        { LedId.Keyboard_2, (1, 2) },
+        { LedId.Keyboard_3, (1, 3) },
+        { LedId.Keyboard_4, (1, 4) },
+        { LedId.Keyboard_5, (1, 5) },
+        { LedId.Keyboard_6, (1, 6) },
+        { LedId.Keyboard_7, (1, 7) },
+        { LedId.Keyboard_8, (1, 8) },
+        { LedId.Keyboard_9, (1, 9) },
+        { LedId.Keyboard_0, (1, 10) },
+        { LedId.Keyboard_MinusAndUnderscore, (1, 11) },
+        { LedId.Keyboard_EqualsAndPlus, (1, 12) },
+        { LedId.Keyboard_Backspace, (1, 13) },
+
+        { LedId.Keyboard_Tab, (2, 0) },
+        { LedId.Keyboard_Q, (2, 1) },
+        { LedId.Keyboard_W, (2, 2) },
+        { LedId.Keyboard_E, (2, 3) },
+        { LedId.Keyboard_R, (2, 4) },
+        { LedId.Keyboard_T, (2, 5) },
+        { LedId.Keyboard_Y, (2, 6) },
+        { LedId.Keyboard_U, (2, 7) },
+        { LedId.Keyboard_I, (2, 8) },
+        { LedId.Keyboard_O, (2, 9) },
+        { LedId.Keyboard_P, (2, 10) },
+        { LedId.Keyboard_BracketLeft, (2, 11) },
+        { LedId.Keyboard_BracketRight, (2, 12) },
+        { LedId.Keyboard_Backslash, (2, 13) },
+
+        { LedId.Keyboard_CapsLock, (3, 0) },
+        { LedId.Keyboard_A, (3, 1) },
+        { LedId.Keyboard_S, (3, 2) },
+        { LedId.Keyboard_D, (3, 3) },
+        { LedId.Keyboard_F, (3, 4) },
+        { LedId.Keyboard_G, (3, 5) },
+        { LedId.Keyboard_H, (3, 6) },
+        { LedId.Keyboard_J, (3, 7) },
+        { LedId.Keyboard_K, (3, 8) },
+        { LedId.Keyboard_L, (3, 9) },
+        { LedId.Keyboard_SemicolonAndColon, (3, 10) },
+        { LedId.Keyboard_ApostropheAndDoubleQuote, (3, 11) },
+        { LedId.Keyboard_NonUsTilde, (3, 12) },
+        { LedId.Keyboard_Enter, (3, 13) },
+
+        { LedId.Keyboard_LeftShift, (4, 0) },
+        { LedId.Keyboard_NonUsBackslash, (4, 1) },
+        { LedId.Keyboard_Z, (4, 2) },
+        { LedId.Keyboard_X, (4, 3) },
+        { LedId.Keyboard_C, (4, 4) },
+        { LedId.Keyboard_V, (4, 5) },
+        { LedId.Keyboard_B, (4, 6) },
+        { LedId.Keyboard_N, (4, 7) },
+        { LedId.Keyboard_M, (4, 8) },
+        { LedId.Keyboard_CommaAndLessThan, (4, 9) },
+        { LedId.Keyboard_PeriodAndBiggerThan, (4, 10) },
+        { LedId.Keyboard_SlashAndQuestionMark, (4, 11) },
+        { LedId.Keyboard_RightShift, (4, 13) },
+
+        { LedId.Keyboard_LeftCtrl, (5, 0) },
+        { LedId.Keyboard_LeftGui, (5, 1) },
+        { LedId.Keyboard_LeftAlt, (5, 2) },
+        { LedId.Keyboard_Custom1, (5, 4) },
+        { LedId.Keyboard_Custom2, (5, 5) },
+        { LedId.Keyboard_Space, (5, 6) },
+        { LedId.Keyboard_Custom3, (5, 7) },
+        { LedId.Keyboard_Custom4, (5, 8) },
+        { LedId.Keyboard_RightAlt, (5, 10) },
+        { LedId.Keyboard_Application, (5, 11) },
```

Comment:
```suggestion
        { LedId.Keyboard_RightGui, (5, 11) },
```

Reply:
This is a windows/apple/"whatever it is on linux"-key isn't it?

Reply:
https://uploads-ssl.webflow.com/612ca8920bc3a648125ffac3/615c0edf7dba264b58e21de0_RGB-top-min.png

Reply:
That's the official default layout.

Reply:
oh wait this is 60HE... then its right alt, application, right ctrl, function

![](https://uploads-ssl.webflow.com/612ca8920bc3a648125ffac3/615c189314611a0fceee2d64_Function-layer-0-min.png)